<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lawyers</class>
 <widget class="QMainWindow" name="Lawyers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lawyers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>事务案件</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pictures/file.png</normaloff>pictures/file.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>用户信息</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pictures/user.png</normaloff>pictures/user.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>庭审信息</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pictures/clock.png</normaloff>pictures/clock.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>退出</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pictures/return.png</normaloff>pictures/return.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
